plugintoolbox/dropper_script_template.py

```diff
@@ -1,6 +1,4 @@
-from pathlib import PurePath
-
-DROPPER_SCRIPT = """#!/bin/bash
+DROPPER_SCRIPT_TEMPLATE = b"""#!/bin/bash
```

Comment:
What about 
```suggestion
AGENT_BINARY_WRAPPER_SCRIPT_TEMPLATE = b"""#!/bin/bash
```

This would better match the name given to the parameter in the registrar (https://github.com/guardicode/agentpluginapi/pull/8).